packages/core/src/App/Components/Elements/Modals/DenialOfServiceModal/denial-of-service.jsx

```diff
@@ -7,9 +7,9 @@ import { website_name } from 'App/Constants/app-config';
 
 const DenialOfServiceModal = ({ disableApp, enableApp, is_loading, is_visible, onCancel, onConfirm }) => (
     <Dialog
-        title={localize('Whoops!')}
-        confirm_button_text={localize('Continue with Demo Account')}
-        cancel_button_text={localize('Back to main website')}
+        title={localize("That's not ready yet!")}
+        confirm_button_text={localize('Stay on {{website_name}}.app', { website_name })}
```

Comment:
Can we use `website_domain` instead of `website_name` so we don't have to hardcode `.app`?